articles.tex

```diff
@@ -680,4 +680,36 @@
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately.
```

Comment:
This sentence implies that the amendment expires both in the future and as soon as it comes into effect.

Reply:
I would suggest, "Will expire after that time"